examples/inverse/plot_use_sourcemorph.py

```diff
@@ -0,0 +1,178 @@
+raw.info['bads'] = ['MEG 2443', 'EEG 053']  # 2 bads channels
+events = mne.read_events(event_fname)
+
+# Set up pick list: EEG + MEG - bad channels (modify to your needs)
+picks = mne.pick_types(raw.info, meg=True, eeg=False, stim=True, eog=True,
+                       exclude='bads')
+
+# Pick the channels of interest
+raw.pick_channels([raw.ch_names[pick] for pick in picks])
+
+# Re-normalize our empty-room projectors, so they are fine after subselection
+raw.info.normalize_proj()
+
+# Read epochs
+epochs = mne.Epochs(raw, events, event_id, tmin, tmax,
+                    baseline=(None, 0), preload=True, proj=True,
+                    reject=dict(grad=4000e-13, mag=4e-12, eog=150e-6))
+evoked = epochs.average()
+
+# Read forward model
+forward = mne.read_forward_solution(fname_fwd)
+forward_surf = mne.convert_forward_solution(forward, surf_ori=True)
+
+# Compute regularized noise and data covariances
+noise_cov = mne.compute_covariance(epochs, tmin=tmin, tmax=0, method='shrunk')
+data_cov = mne.compute_covariance(epochs, tmin=0.04, tmax=0.15,
+                                  method='shrunk')
+
+# Compute unit-noise-gain beamformer with whitening of the leadfield and
+# data (enabled by passing a noise covariance matrix)
+filters = make_lcmv(evoked.info, forward_surf, data_cov, reg=0.05,
+                    noise_cov=noise_cov, pick_ori='max-power',
+                    weight_norm='nai')
+
+# Apply this spatial filter to source-reconstruct the evoked data
+stc_surf = apply_lcmv(evoked, filters, max_ori_out='signed')
+
+# To save memory
+stc_surf.crop(0.087, 0.087)
+
+# Compute VolSourceEstimate
+fname_fwd = data_path + '/MEG/sample/sample_audvis-meg-vol-7-fwd.fif'
+
+# Read forward model
+forward_vol = mne.read_forward_solution(fname_fwd)
+
+filters = make_lcmv(evoked.info, forward_vol, data_cov, reg=0.05,
+                    noise_cov=noise_cov, pick_ori='max-power',
+                    weight_norm='nai')
+
+# Apply this spatial filter to the evoked data.
+stc_vol = apply_lcmv(evoked, filters, max_ori_out='signed')
+
+# To save memory
+stc_vol.crop(0.087, 0.087)
+
+###############################################################################
+# Morph SourceEstimate
+
+# Initialize morpher for SourceEstimate
+src_surf = forward_surf['src']
+source_morph_surf = SourceMorph(src_surf, subjects_dir=subjects_dir)
+
+# Save and load SourceMorph if desired
+# source_morph_surf.save('surf')
+# source_morph_surf = mne.read_source_morph('surf-morph.h5')
+
+# Morph data
+np.abs(stc_surf.data, out=stc_surf.data)  # for plotting
+stc_surf_fsaverage = source_morph_surf(stc_surf)
+
+###############################################################################
+# Morph VolSourceEstimate
+
+# Initialize morpher for VolSourceEstimate
+src_vol = forward_vol['src']
+src_vol[0]['subject_his_id'] = 'sample'
+source_morph_vol = SourceMorph(src_vol, subjects_dir=subjects_dir,
+                               grid_spacing=(5., 5., 5.))
```

Comment:
why is the subject not specified in the volume source space?
I don't like exposing low level code like `src_vol[0]['subject_his_id'] = 'sample'` in examples.
Can we had a subject parameter to SourceMorph that is None by default and leads to error if 
subject name cannot be found in src?

Reply:
I would also explicit here that subject_to is fsaverage even if it's the default parameter